js/app/widgets/dynamicLogo.js

```diff
@@ -135,6 +152,12 @@ const DynamicLogo = new Knowledge.Class({
     },
 
     _update_custom_style: function () {
+        let width = EosKnowledgePrivate.style_context_get_custom_int(this.get_style_context(), 'max-width');
```

Comment:
Factor out `let context = this.get_style_context();`, this function will be called quite often, potentially on every mouse movement